notification/src/config/config.js

```diff
@@ -53,6 +53,15 @@ function getAllAdyenMerchantAccounts() {
   return Object.keys(config.adyen)
 }
 
+function getPaymentMethodsToNamesConfig() {
+  return {
+    scheme: 'Credit Card',
+    pp: 'PayPal',
+    klarna: 'Klarna',
```

Comment:
why `gpay: 'Google Pay'` is not added here?

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/699/commits/8667921cf24dfd8712ceda431a0bf46a00e013cc